extern/vecmem/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building VecMem as part of the TRACCC project" )
+
+# Declare where to get VecMem from.
+set( TRACCC_VECMEM_SOURCE
+   "URL;https://github.com/acts-project/vecmem/archive/refs/tags/v0.6.0.tar.gz;URL_MD5;d741067ff148d2df3c4e6d4587d65fdc"
+   CACHE STRING "Source for VecMem, when built as part of this project" )
+mark_as_advanced( TRACCC_VECMEM_SOURCE )
+FetchContent_Declare( VecMem ${TRACCC_VECMEM_SOURCE} )
+
+# Options used in the build of VecMem.
+set( VECMEM_BUILD_TESTING FALSE CACHE BOOL
+   "Turn off the build of the VecMem unit tests" )
```

Comment:
Same thing as before, maybe building the tests should be disabled by default on the vecmem side.

Reply:
I would be in favour of that. However, that is not really relevant right now. We can take that discussion to the relevant projects.